client/src/components/bpmn.modeler.component.jsx

```diff
@@ -1,69 +1,104 @@
-import React, {Component} from "react";
-import BpmnModeler from "bpmn-js/lib/Modeler";
-import "bpmn-js/dist/assets/diagram-js.css";
-import "bpmn-font/dist/css/bpmn-embedded.css";
-import {emptyBpmn} from "../assets/empty.bpmn";
-import propertiesPanelModule from "bpmn-js-properties-panel";
-import propertiesProviderModule from "bpmn-js-properties-panel/lib/provider/camunda";
-import camundaModdleDescriptor from "camunda-bpmn-moddle/resources/camunda";
+import React, { Component } from 'react';
+import BpmnModeler from 'bpmn-js/lib/Modeler';
+import 'bpmn-js/dist/assets/diagram-js.css';
+import 'bpmn-font/dist/css/bpmn-embedded.css';
+import { emptyBpmn } from '../assets/empty.bpmn';
+import propertiesPanelModule from 'bpmn-js-properties-panel';
+import propertiesProviderModule from 'bpmn-js-properties-panel/lib/provider/camunda';
+import camundaModdleDescriptor from 'camunda-bpmn-moddle/resources/camunda';
+const parser = require('../parser.js');
+var convert = require('xml-js');
 
 class BpmnModelerComponent extends Component {
```

Comment:
maybe use default export here already? This has been done in other files as well, so I would at least use a consistent scheme. Let me know on what you decide

Reply:
We will use default export always at the end of the file.